src/BookNavigator/sharing.js

```diff
@@ -12,16 +12,16 @@ export default class SharingProvider {
     const creatorToUse = Array.isArray(creator) ? creator[0] : creator;
     const subPrefix = bookreader.options.subPrefix || '';
     const label = `Share this book`;
-    this.icon = html`<ia-icon-share style="width: var(--iconWidth); height: var(--iconHeight);"></ia-icon-share>`;
+    this.icon = html`${iauxShareIcon}`;
     this.label = label;
     this.id = 'share';
-    this.component = html`<ia-sharing-options
+    this.component = html`<iaux-sharing-options
```

Comment:
I think we might want to standardize on a naming scheme for these ; I think `<iaux-sharing-panel>`, `<iaux-files-panel>`, etc might be the way to go! I generally change the prefix to be who 'owns' the components, so BookReader sub-components start with `br` (eg `<br-mode-1up>`). So maybe we could `<in-sharing-panel>` for `itemnav`?

Reply:
yes I like that. updated to `<iaux-in-share-panel>` - keeping `iaux-` as latest preferred web component prefix